### PR TITLE
Arsenal tech overhaul

### PR DESCRIPTION
**Pull request by Stop-Signs**
## About the PR
Merges some of the redundant techs, reduces the amount of content locked behind getting t3 arsenal. adjusts some pricing and moves the incendiary ammo and uranium ammo techs to tier 2. 

## Why / Balance
The 2 laser cannons being single item techs really wasnt doing sec any favors, neither was having the pdw and ik60 cost 20k combined. merging them with a slight price increase should overall be better for the game. moving the advanced munitions to t2 is kind of just where i thought they fit better, they are t1's that both cost a large amount of money, and they aren't exactly basic things for sec. Svallins being moved to t2 shouldn't cause many issues as the gun sucks and sec can just confiscate them(if needed I can move the recipe location, remove the tech alltogether, or move it back to t3) Basic shuttle armament is now a t1, but you have to get explosive tech and salv weaponry first. advanced shuttle armament is now a t2. hopefully with events involving shuttles coming soon, this will allow sec to be able to have those oh so fun shuttle battles. Advanced laser pistol is remaining a t3 for obvious reasons.

## Technical details
Comments were left on everything (i hope) still working on that new t3, will update when I do.

## Media
image
## Requirements

- [x ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ x] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

:cl:
- add: New Sec melee for epi to research!
- tweak: Merged several arsenal techs to better fit the state of epi, as well as changing the tiering and prices.

### CODE REVIEW

**Comment by Stop-Signs** [comment]
why did it request velcro?? no offense

**Comment by Stop-Signs** [comment]
i have a feeling that these errors are not fake :3

File: Resources/Prototypes/Recipes/Lathes/security.yml [hunk 1]
```diff
@@ -657,6 +657,16 @@
     Plastic: 100
     Glass: 20
 
+    - type: latheRecipe
```
Comment (IamVelcroboy) [inline Resources/Prototypes/Recipes/Lathes/security.yml]: Indented too far

File: Resources/Prototypes/Research/arsenal.yml [hunk 2]
```diff
@@ -157,57 +136,69 @@
...
-  discipline: Arsenal
-  tier: 2
-  cost: 10000
-  recipeUnlocks:
-  - WeaponXrayCannon
-
-- type: technology
-  id: BasicShuttleArmament
-  name: research-technology-basic-shuttle-armament
+  - type: technology
```
Comment (IamVelcroboy) [inline Resources/Prototypes/Research/arsenal.yml]: Another over-indent here

File: Resources/Prototypes/Research/arsenal.yml [hunk 3]
```diff
@@ -216,23 +207,41 @@
...
+  icon:
+    sprite: Objects/Weapons/Guns/Battery/advancedlasergun.rsi
+    state: icon
+  discipline: Arsenal
+  tier: 3
+  cost: 15000
+  recipeUnlocks:
+  - WeaponAdvancedLaser
+
+
```
Comment (IamVelcroboy) [inline Resources/Prototypes/Research/arsenal.yml]: Only leave 1 extra line at the end of the file

File: Resources/Prototypes/Recipes/Lathes/security.yml [hunk 4]
```diff
@@ -657,6 +657,16 @@
...
+- type: latheRecipe
+  parent: BaseWeaponRecipe
+  id: AdvancedTruncheon
+  result: AdvancedTruncheon
+  completetime: 5
+  Materials:
+    Steel: 1000
+    Plastic: 800
+  
+
```
Comment (IamVelcroboy) [inline Resources/Prototypes/Recipes/Lathes/security.yml]: remove a line here

File: Resources/Prototypes/Research/arsenal.yml [hunk 5]
```diff
@@ -75,29 +76,7 @@
...
-  - MagazineBoxPistolUranium
-  - MagazineBoxMagnumUranium
-  - MagazineBoxLightRifleUranium
-  - MagazineBoxRifleUranium
-  - BoxShotgunUranium
-  # DeltaV - .38 special uranium ammo - Adds .38 special uranium ammo to the research tree
-  - SpeedLoaderSpecialUranium
-  - MagazineBoxSpecialUranium
-  # End of modified code
+
```
Comment (IamVelcroboy) [inline Resources/Prototypes/Research/arsenal.yml]: remove a line here

**Comment by MilonPL** [comment]
@IamVelcroboy for the love of all that is holy why did you submit this as 5 individual reviews

**Comment by Stop-Signs** [comment]
I HOPE that fixed it, we shall see

File: Resources/Prototypes/DeltaV/Research/arsenal.yml [hunk 6]
```diff
@@ -32,24 +32,38 @@
...
 
-- type: technology
-  id: Advanced Laser Manipulation
-  name: research-technology-advance-laser
+- type:  technology
+  id: RobustMelee
+  name: research-technology-robust-melee
   icon:
-    sprite: DeltaV/Objects/Weapons/Guns/Battery/energygun_carbine.rsi
+    sprite: Objects/Weapons/Melee/Advanced trunch
```
Comment (Radezolid) [inline Resources/Prototypes/DeltaV/Research/arsenal.yml]: If you want the truncheon icon this should be the path.
`Resources\Textures\Objects\Weapons\Melee\truncheon.rsi`

Comment (Stop-Signs) [reply Resources/Prototypes/DeltaV/Research/arsenal.yml]: i made my own sprites, but didnt label the file as RSI. is that an issue?

**Comment by IamVelcroboy** [comment]
> @IamVelcroboy for the love of all that is holy why did you submit this as 5 individual reviews

🤷‍♂️ I was on my phone, looking through the file changes

File: Resources/Prototypes/DeltaV/Research/arsenal.yml (hunk 6 above)
Comment (Radezolid) [reply Resources/Prototypes/DeltaV/Research/arsenal.yml]: I mean, it's literally why the YAML linter is dying.
` [ERRO] go.comp.sprite: Unable to load RSI '/Textures/Objects/Weapons/Melee/Advanced trunch'.`
Edit: Also remove the space, use _ instead.

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/advanced_truncheon.yml
Comment (Radezolid) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/advanced_truncheon.yml:1]: This file should go in the DeltaV folder

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/advanced_truncheon.yml [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+#  - type: MeleeStaminaCost
+#    swing: 10
+#    wieldCoefficient: 0.35 #if wielded you will only consume 3.5 stam its a weapon after all
+  - type: Wieldable
+  - type: IncreaseDamageOnWield
+    damage:
+      types:
+        Blunt: 14
+  - type: Clothing
+    sprite: Objects/Weapons/Melee/Advanced trunch
```
Comment (Radezolid) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/advanced_truncheon.yml]: Need to update the sprite path here.

File: Resources/Prototypes/DeltaV/Research/arsenal.yml [hunk 8]
```diff
@@ -32,24 +32,38 @@
   id: EnergyGunsAdvanced
   name: research-technology-energy-gun-advance
   icon:
-    sprite: DeltaV/Objects/Weapons/Guns/Battery/energygun_pistol.rsi
+    sprite: DeltaV/Objects/Weapons/Guns/Battery/energygun_carbine.rsi #changed sprite to ik-60
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Research/arsenal.yml]: dont need this comment

File: Resources/Locale/en-US/deltav/research/technologies.ftl [hunk 9]
```diff
@@ -4,3 +4,4 @@ research-technology-exotic-ammunition = Exotic Ammunition
 research-technology-energy-gun = Energy Guns
 research-technology-energy-gun-advance = Advanced Energy Manipulation
 research-technology-advance-laser = Advanced Laser Manipulation
+research-technology-robust-melee = Robust Melee
```
Comment (deltanedas) [inline Resources/Locale/en-US/deltav/research/technologies.ftl]: file needs to end in a newline

File: Resources/Prototypes/DeltaV/Research/arsenal.yml [hunk 10]
```diff
@@ -32,24 +32,38 @@
...
-  cost: 7500
+  cost: 12500
   recipeUnlocks:
   - WeaponEnergyGunPistol
+  - WeaponGunLaserCarbineAutomatic
 
-- type: technology
-  id: Advanced Laser Manipulation
-  name: research-technology-advance-laser
+- type:  technology
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Research/arsenal.yml]: pro space

File: Resources/Prototypes/DeltaV/Research/arsenal.yml [hunk 11]
```diff
@@ -32,24 +32,38 @@
...
+#  id: Advanced Laser Manipulation
+#  name: research-technology-advance-laser
+#  icon:
+#    sprite: DeltaV/Objects/Weapons/Guns/Battery/energygun_carbine.rsi
+#    state: icon
+#  discipline: Arsenal
+#  tier: 2
+ # cost: 12500
+#  recipeUnlocks:
+#  - WeaponGunLaserCarbineAutomatic
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Research/arsenal.yml]: just remove dont comment it out

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/advanced_truncheon.yml
Comment (deltanedas) [reply Resources/Prototypes/Entities/Objects/Weapons/Melee/advanced_truncheon.yml:1]: it should, and also inherit truncheon

File: Resources/Textures/Objects/Weapons/Melee/meta.json [hunk 12]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (deltanedas) [inline Resources/Textures/Objects/Weapons/Melee/meta.json]: delete

File: Resources/Prototypes/Research/arsenal.yml
Comment (deltanedas) [inline Resources/Prototypes/Research/arsenal.yml:1]: you reordered half the techs in this file for no reason which will make 10000 conflicts and makes it infinitely harder to review

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/advanced_truncheon.yml [hunk 13]
```diff
@@ -0,0 +1,33 @@
+- type: entity
+  name: Advanced Truncheon
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/advanced_truncheon.yml]: advanced truncheon

also parent -> id -> name -> description, reorder the fields

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Textures/Objects/Weapons/Melee/meta.json
Comment (deltanedas) [inline Resources/Textures/Objects/Weapons/Melee/meta.json:1]: bro delete the file :godmode:

Comment (Stop-Signs) [reply Resources/Textures/Objects/Weapons/Melee/meta.json:1]: without the fi------ nevermind i see the issue

**Comment by Stop-Signs** [comment]
@MilonPL its done

**Comment by deltanedas** [comment]
this pr still mangles upstreams arsenal.yml

**Comment by Stop-Signs** [comment]
> this pr still mangles upstreams arsenal.yml

the techs are laid in order of tier, i just moved them into those areas, i can move them back but It wouldnt be organized the same way as upstream

**Comment by Stop-Signs** [comment]
this pr still mangles upstreams arsenal.yml


what would yall prefer me do. i could comment out the techs and copy paste them onto the delta tech files. or i could move everything back where it was and have techs be in the inappropriate tier-section, but still be in place for upstream

**Comment by deltanedas** [comment]
the sections dont matter, conflicts do

**Comment by Stop-Signs** [comment]
> the sections dont matter, conflicts do

should be fixed @deltanedas

**Comment by Colin-Tel** [comment]
image
Please reduce your changelog to just one or two lines.

File: Resources/Prototypes/DeltaV/Research/arsenal.yml [hunk 14]
```diff
@@ -32,24 +32,23 @@
...
-  name: research-technology-advance-laser
+  id: RobustMelee
+  name: research-technology-robust-melee
   icon:
-    sprite: DeltaV/Objects/Weapons/Guns/Battery/energygun_carbine.rsi
+    sprite: DeltaV/Objects/Weapons/Melee/advanced_truncheon.rsi
     state: icon
   discipline: Arsenal
-  tier: 2
+  tier: 3
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Research/arsenal.yml:51]: i still dont see non antags getting tier 3 arsenal for funny kanabou

**Review by deltanedas — DISMISSED** [review]
im happy codewise, just above

File: Resources/Prototypes/Research/arsenal.yml [hunk 15]
```diff
@@ -21,7 +21,7 @@
     sprite: Objects/Weapons/Guns/Ammunition/Boxes/pistol.rsi
     state: incendiarydisplay
   discipline: Arsenal
-  tier: 1
+  tier: 2 #Delta-V moved to t2 from t1
```
Comment (MilonPL) [inline Resources/Prototypes/Research/arsenal.yml]: Remove the hyphen in `Delta-V`.

File: Resources/Prototypes/Research/arsenal.yml [hunk 16]
```diff
@@ -82,7 +82,7 @@
     sprite: Objects/Materials/Sheets/other.rsi
     state: uranium
   discipline: Arsenal
-  tier: 1
+  tier: 2 #Delta-V moved to t2 from t1
```
Comment (MilonPL) [inline Resources/Prototypes/Research/arsenal.yml]: Remove the hyphen in `Delta-V`.

**Review by MilonPL — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Recipes/Lathes/security.yml
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Recipes/Lathes/security.yml:1]: newline at the end of file

File: Resources/Prototypes/Research/arsenal.yml [hunk 17]
```diff
@@ -21,7 +21,7 @@
     sprite: Objects/Weapons/Guns/Ammunition/Boxes/pistol.rsi
     state: incendiarydisplay
   discipline: Arsenal
-  tier: 1
+  tier: 2 #DeltaV moved to t2 from t1
```
Comment (MilonPL) [inline Resources/Prototypes/Research/arsenal.yml]: trailing space
```suggestion
  tier: 2 # DeltaV moved to t2 from t1
```

File: Resources/Prototypes/Research/arsenal.yml [hunk 18]
```diff
@@ -82,7 +82,7 @@
     sprite: Objects/Materials/Sheets/other.rsi
     state: uranium
   discipline: Arsenal
-  tier: 1
+  tier: 2 #DeltaV moved to t2 from t1
```
Comment (MilonPL) [inline Resources/Prototypes/Research/arsenal.yml]: trailing space

**Review by MilonPL — CHANGES_REQUESTED** [review]

**Comment by Stop-Signs** [comment]
ubuntu error is fake, this is ready to merge

File: Resources/Prototypes/Research/arsenal.yml (hunk 18 above)
Comment (MilonPL) [inline Resources/Prototypes/Research/arsenal.yml]: trailing space still.

File: Resources/Prototypes/DeltaV/Research/arsenal.yml [hunk 19]
```diff
@@ -32,24 +32,23 @@
   id: EnergyGunsAdvanced
   name: research-technology-energy-gun-advance
   icon:
-    sprite: DeltaV/Objects/Weapons/Guns/Battery/energygun_pistol.rsi
+    sprite: DeltaV/Objects/Weapons/Guns/Battery/energygun_carbine.rsi
```
Comment (MilonPL) [inline Resources/Prototypes/DeltaV/Research/arsenal.yml]: remove the space after `energygun_carbine.rsi`

**Review by MilonPL — CHANGES_REQUESTED** [review]

**Review by MilonPL — APPROVED** [review]

**Pull request merged**